lib/datadog/core/diagnostics/environment_logger.rb

```diff
@@ -257,7 +257,7 @@ def collect!(transport_responses)
         private
 
         def instrumented_integrations
-          Datadog.configuration.instrumented_integrations
+          Datadog.configuration.tracing.instrumented_integrations
```

Comment:
Having tracing be the global manager of integrations feels awkward to me, but I guess this is an intermediate step towards each component have its own integration management (which would be an important step towards appsec using the same integration and configuration code as other components.

Reply:
Yeah, it's intermediate. They should definitely manage their own.